server/utilities/handle-csv.js

```diff
@@ -0,0 +1,33 @@
+const Customer = require('../models/Customer')
+const fs = require('fs')
+const csv = require('fast-csv')
+
+const handleCSV = (path, req, res) => {
+  let customers = []
+
+  fs.createReadStream(path)
+    .pipe(csv.parse({ headers: ['email', 'firstName', 'lastName', 'phone', 'city', 'state', 'lastOrderPrice', 'lastOrderDate'], renameHeaders: true }))
+    .on('error', (error) => {
+      throw error.message
+    })
+    .on('data', (row) => {
+      customers.push(row)
+    })
+    .on('end', () => {
+      Customer.bulkCreate(customers)
+        .then(() => {
+          return res.status(200).send({
```

Comment:
nit: if there is only one line inside a function with a return you can replace it for 
```
.then(() => (res.status(200).send({message: 'Uploaded...'})))
```
Check where else you can apply this.